CS3230Project/DAL/Nurses/NurseDal.cs

```diff
@@ -0,0 +1,128 @@
+            var nurseIDOrdinal = reader.GetOrdinal("nurseId");
+            var employeeIDOrdinal = reader.GetOrdinal("employeeId");
+            var firstNameOrdinal = reader.GetOrdinal("firstName");
+            var lastNameOrdinal = reader.GetOrdinal("lastName");
+            var dateOfBirthOrdinal = reader.GetOrdinal("dateOfBirth");
+            var genderOrdinal = reader.GetOrdinal("gender");
+            var phoneOrdinal = reader.GetOrdinal("phone");
+            var addressOneOrdinal = reader.GetOrdinal("addressOne");
+            var addressTwoOrdinal = reader.GetOrdinal("addressTwo");
+            var cityOrdinal = reader.GetOrdinal("city");
+            var stateOrdinal = reader.GetOrdinal("state");
+            var zipcodeOrdinal = reader.GetOrdinal("zipcode");
+            var usernameOrdinal = 2; //will be removed after merge, I think.
+
+            while (reader.Read())
+            {
+                nurse = new Nurse(
+                    reader.GetInt32(nurseIDOrdinal),
+                    reader.GetFieldValueCheckNull<string>(firstNameOrdinal),
+                    reader.GetFieldValueCheckNull<string>(lastNameOrdinal),
+                    reader.GetFieldValueCheckNull<DateTime>(dateOfBirthOrdinal),
+                    reader.GetFieldValueCheckNull<string>(genderOrdinal),
+                    reader.GetFieldValueCheckNull<string>(phoneOrdinal),
+                    reader.GetFieldValueCheckNull<string>(addressOneOrdinal),
+                    reader.GetFieldValueCheckNull<string>(addressTwoOrdinal),
+                    reader.GetFieldValueCheckNull<string>(cityOrdinal),
+                    reader.GetFieldValueCheckNull<string>(stateOrdinal),
+                    reader.GetFieldValueCheckNull<string>(zipcodeOrdinal),
+                    reader.GetFieldValueCheckNull<string>(usernameOrdinal));
+            }
+
+            return nurse;
+        }
+
+        private static List<Nurse> createNurses(MySqlCommand command)
+        {
+            var nurses = new List<Nurse>();
+
+            using var reader = command.ExecuteReader();
+            var nurseIDOrdinal = reader.GetOrdinal("nurseId");
+            var employeeIDOrdinal = reader.GetOrdinal("employeeId");
+            var firstNameOrdinal = reader.GetOrdinal("firstName");
+            var lastNameOrdinal = reader.GetOrdinal("lastName");
+            var dateOfBirthOrdinal = reader.GetOrdinal("dateOfBirth");
+            var genderOrdinal = reader.GetOrdinal("gender");
+            var phoneOrdinal = reader.GetOrdinal("phone");
+            var addressOneOrdinal = reader.GetOrdinal("addressOne");
+            var addressTwoOrdinal = reader.GetOrdinal("addressTwo");
+            var cityOrdinal = reader.GetOrdinal("city");
+            var stateOrdinal = reader.GetOrdinal("state");
+            var zipcodeOrdinal = reader.GetOrdinal("zipcode");
+            var usernameOrdinal = 2; //will be removed after merge, I think.
```

Comment:
This should either be removed or changed to an actual username value

Reply:
Will be fixed. Waiting on merge.

Reply:
fixed